linode/manager
# Table consistency tickets

## Description

Make sure all tables are referencing the right `TableRow` component and define the height for table rows

## How to test

Check the following tables to make sure the color specs and row height (headers are 46px, rows are 40px) are consistent in dark mode 
- Linode Details Activity Feed table
- Invoice table

Check the header hover color spec for the Linodes and StackScripts Landing table in dark mode
